pypowsybl/sensitivity_analysis.py

```diff
@@ -56,8 +56,8 @@ def set_factor_matrix(self, branches_ids: List[str], injections_or_transformers_
         self.branches_ids = branches_ids
         self.injections_or_transformers_ids = injections_or_transformers_ids
 
-    def run_dc(self, network: Network, parameters: Parameters = Parameters()) -> SensitivityAnalysisResult:
-        return SensitivityAnalysisResult(_pypowsybl.run_sensitivity_analysis(self.ptr, network.ptr, parameters),
+    def run_dc(self, network: Network, parameters: Parameters = Parameters(), provider = 'OpenLoadFlow') -> SensitivityAnalysisResult:
```

Comment:
Actually I don't think it will work !
The sensitivity analysis is named OpenSensitivityAnalysis